apps/web-mzima-client/src/app/settings/surveys/survey-task/survey-task.component.html

```diff
@@ -83,16 +83,34 @@ <h2>
           [data-qa]="'field-' + field"
         >
           <div class="field-item__main">
-            <mat-icon
-              cdkDragHandle
-              class="field-item__non-drag-handle"
-              svgIcon="drag-handle"
-            ></mat-icon>
-            <span class="field-item__title" [data-qa]="'field-name'">{{
-              isDefaultLanguageSelected
-                ? field.label
-                : field.translations[selectLanguageCode]?.label || field.label
-            }}</span>
+            <div
+              class="field-item__main-inner"
+              (mouseenter)="trackHoverEvent($event, field)"
+              (mouseleave)="trackHoverEvent($event, field)"
+            >
+              <mat-icon
+                cdkDragHandle
+                class="field-item__non-drag-handle"
+                svgIcon="drag-handle"
+              ></mat-icon>
+              <span class="field-item__title" [data-qa]="'field-name'">{{
+                isDefaultLanguageSelected
+                  ? field.label
+                  : field.translations[selectLanguageCode]?.label || field.label
+              }}</span>
+              <mzima-client-button
+                *ngIf="fieldId === field.id && fieldHover"
+                fill="clear"
+                color="secondary"
+                [iconOnly]="true"
+                class="tooltip-btn"
+                tabindex="-1"
+                matTooltipPosition="right"
+                matTooltip="{{ field.label_original }}"
+              >
```

Comment:
We don't handle this in other places so it might be time for a separate ticket for making the tooltip accessible.

Reply:
There is this part too... 👍🏼